content/en/docs/faq/_index.md

```diff
@@ -20,19 +20,50 @@ face:
 
 This is a feature in cert-manager starting in `v0.16` using the kubectl plugin. More information can be found on [the renew command's page](../usage/kubectl-plugin/#renew)
 
-### How can I see all the historic events related to a certificate object ?
+### Why isn't my root certificate in my issued Secret's `tls.crt`?
+
+Occasionally, people work with systems which have made a flawed choice regarding TLS chains. The [TLS spec](https://datatracker.ietf.org/doc/html/rfc5246#section-7.4.2)
+has the following section for the "Server Certificate" section of the TLS handshake:
+
+> This is a sequence (chain) of certificates.  The sender's
+> certificate MUST come first in the list.  Each following
+> certificate MUST directly certify the one preceding it.  Because
+> certificate validation requires that root keys be distributed
+> independently, the self-signed certificate that specifies the root
+> certificate authority MAY be omitted from the chain, under the
+> assumption that the remote end must already possess it in order to
+> validate it in any case.
+
+In a standard, secure and correctly configured TLS environment, adding a root certificate to the chain is almost entirely _pure waste_.
```

Comment:
this reads better for sure, thanks. I've added a "almost always" before "unnecessary" because I can come up with very convoluted scenarios where it might not be entirely wasteful, such as:

Having the root in the chain theoretically allows a validator to more quickly check if the root appears in their trust store... but only really if they were intentionally trying to create pathologically evil trust stores with huge numbers of duplicated certificates :joy: